modules/tinymce/src/core/main/ts/undo/UndoManagerTypes.ts

```diff
@@ -43,4 +43,4 @@ export type Index = Cell<number>;
 
 export type Locks = Cell<number>;
 
-export type UndoBookmark = Cell<Optional<Bookmark>>;
+export type UndoBookmark = Singleton.Value<Bookmark>;
```

Comment:
FYI @TheSpyder this will change the type of object being passed to RTC. However, I had a look and RTC doesn't implement `add` or `beforeChange` yet so it shouldn't be impacted by this. If it will cause issues let me know and I'll revert it back.

Reply:
If you look closely the type is only used in `RtcAdaptor`, which is a copy of the API types for the purposes of making runtime adaptors. The actual RTC api is interface `RtcRuntimeApi` so yes the value is never passed to the plugin (because, as you found, we haven't implemented the entire undo manager API in the alt-model).

Reply:
Right, I was looking at the adaptor instead. Thanks for the clarification!